Update to beta note on Session Replay SDK Docs (#6017)
getsentry/sentry-docs: src/includes/beta-note-session-replay.mdx

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
**Currently, this feature is only supported for browser JavaScript, including these frameworks:**

- Angular
- Ember
- Gatsby

> - Next.js

Author: ```suggestion
```

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
**Currently, this feature is only supported for browser JavaScript, including these frameworks:**

- Angular
- Ember
- Gatsby
- Next.js

> - React

Author: ```suggestion
```

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
**Currently, this feature is only supported for browser JavaScript, including these frameworks:**

- Angular
- Ember
- Gatsby
- Next.js
- React

> - Remix

Author: ```suggestion
```

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
**Currently, this feature is only supported for browser JavaScript, including these frameworks:**

- Angular
- Ember
- Gatsby
- Next.js
- React
- Remix

> - Svelte

Author: ```suggestion
```

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).

> **Currently, this feature is only supported for browser JavaScript, including these frameworks:**

Author: ```suggestion
Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs.
```

- Angular

> - Ember

Author: ```suggestion

```

- Gatsby
- Next.js
- React
- Remix
- Svelte

> - Vue

Author: ```suggestion
```

<Note>

> Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please open a [GitHub issue](https://github.com/getsentry/sentry/issues/new/choose). Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).

Reviewer: Should we ask them to also include a link to the replay and/or the page where they are having issues?

Author: that's a good idea! also realized this beta note is on SDK docs, so going to change the repo link to `sentry-javascript`

> Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please open a [GitHub issue](https://github.com/getsentry/sentry/issues/new/choose). Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).

Author: ```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or mention the Sentry page you were viewing. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```

Author: cc/ @billyvg

Reviewer: ```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or if possible a publicly accessible URL to the page you are attempting to record a replay of. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```

**Currently, this feature is supported for only browser JavaScript, including these frameworks:**

> - Angular

Reviewer 2: do we need to list these frameworks explicitly? as we'll work on other frameworks, there's just not a specific sentry sdk for them.

Author: We have this list on our SDK docs on our product docs - https://docs.sentry.io/product/session-replay/. Considering the volume of inquiries around if we formally support Electron, iOS, React Native, etc. I thought it might be helpful to repeat it here since these docs are viewed often

Maybe we could shorten this to something like - "Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs." instead of individually listing? What do you think?

Reviewer 2: sounds good to me :) may be worth mentioning that it _doesn't_ currently support "Electron, iOS, React Native"

> Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or if possible a publicly accessible URL to the page you are attempting to record a replay of. Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
> 

Reviewer 3: ```suggestion
Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).

```

<Note>

Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new/choose) with a link to a relevant replay or, if possible, a publicly accessible URL to the page you're attempting to record a replay of. You can also email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
**Currently, this feature is only supported for browser JavaScript, including these frameworks:**

> - Angular

Author: ```suggestion
```

> - Gatsby

Author: ```suggestion
```
